premake5.lua

```diff
@@ -616,6 +627,8 @@ standardProject "slang"
     -- `slang.h` header in this prject, so we do that manually here.
     files { "slang.h" }
 
+    files { "source/core/core.natvis" }
```

Comment:
This is good if it works. I did in the past try this, and the project didn't seem to access the natvis files correctly, but the project files seem plausible in the PR.

Reply:
Like you said, the project file seems plausible. I will make sure to double-check that it actually works in the debugger before checking in.

FWIW, I generated these project files using the version of premake checked into `slang-binaries`. Using that one instead of the local copy I had is what gave me plausible (instead of obviously-wrong) lines in the project file for the `.natvis` files.